src/main/scala/tech/cryptonomic/conseil/tezos/DatabaseConversions.scala

```diff
@@ -3,10 +3,15 @@ package tech.cryptonomic.conseil.tezos
 import tech.cryptonomic.conseil.tezos.TezosTypes._
 import tech.cryptonomic.conseil.tezos.FeeOperations._
 import tech.cryptonomic.conseil.util.Conversion
-import tech.cryptonomic.conseil.util.Conversion.Id
+import tech.cryptonomic.conseil.util.Conversion._
 
 object DatabaseConversions {
 
+  import java.sql.Timestamp
+
+  //BEWARE: implicit conversion!!
```

Comment:
please convince me why we need implicit conversion 😄

Reply:
low risk, high gain